MAiD - Case Management App/force-app/main/default/flows/Provincial_and_RX_Flow_for_1633.flow-meta.xml

```diff
@@ -1195,13 +1195,21 @@
                     <stringValue>Discontinuation of Planning: Withdrawn Request</stringValue>
                 </rightValue>
             </conditions>
+            <conditions>
+                <leftValueReference>ListCaseDetail.Type</leftValueReference>
+                <operator>EqualTo</operator>
+                <rightValue>
+                    <stringValue>Discontinuation of Planning: Ineligible</stringValue>
+                </rightValue>
+            </conditions>
             <connector>
                 <targetReference>check_case_type_62</targetReference>
             </connector>
             <label>Check if 1633 is ticked on 1634</label>
         </rules>
     </decisions>
-    <description>Check on Rule 65</description>
+    <description>1633 field check condition is updated..!</description>
+    <environments>Default</environments>
```

Comment:
@AmishMistri: I found this tag getting pulled from our dev environments. Please remove this from the meta file.